dashboard/src/reducers/userProfileReducer.js

```diff
@@ -1,8 +1,15 @@
-import { GET_USER_DETAILS, UPDATE_USER_DETAILS } from "../actions/types";
+import {
+  GET_USER_DETAILS,
+  UPDATE_USER_DETAILS,
+  RESET_DATA,
+  SET_USER_DETAILS,
+} from "../actions/types";
 
 const initialState = {
   userDetails: {},
   userDetails_copy: {},
+  updatedUserDetails: {},
+  isUserDetailsUpdated: false,
```

Comment:
Where is this used?

Reply:
It is used in ProfileComponent/index.jsx to disable/enable the `Save` button.
By checking `updatedUserDetails` is empty the same behaviour can be achieved but it would be a additional load.